### ISSUE TITLE
Redirect printf to other function (like uart) to take log

### ISSUE THREAD

**Issue by phamvantien96** [nemequ/munit#60]
I find a C test framework for embedded, I find this but have a problem with the printf, puts function like a other function in stdio.h. So how can I redirect the printf.

**Comment by codylico** [comment]
The macro `MUNIT_OUTPUT_FILE` might be useful. At a glance, it seems to handle most of the library's main output. (Help message is hard-coded to `printf`, `puts` though.)

**Comment by phamvantien96** [comment]
Yeah, but seem not work at my system when I can't use the fprintf function.

**Comment by strohlaj** [comment]
Isn't this is often something that is provided by the board support package for your embedded system? The BSP that gets generated with my Xilinx Arm Cortex-M processor has a configuration file that indicates what uart the printf function outputs to.
